packages/braintree-integration/src/components/BraintreeAchPaymentForm/BraintreeAchPaymentForm.tsx

```diff
@@ -17,19 +25,33 @@ export interface AddressKeyMap<T = string> {
 }
 
 export interface BraintreeAchPaymentFormProps extends Omit<PaymentMethodProps, 'onUnhandledError'> {
-    mandateText: string;
+    outstandingBalance?: number;
+    storeName?: string;
+    setCurrentMandateText: Dispatch<SetStateAction<string>>;
 }
 
 const BraintreeAchPaymentForm: FunctionComponent<BraintreeAchPaymentFormProps> = ({
     paymentForm: { getFieldValue, setFieldValue, setValidationSchema, isSubmitted, setSubmitted },
-    mandateText,
+    outstandingBalance,
+    storeName,
     checkoutState,
     method,
     language,
+    setCurrentMandateText,
 }) => {
-    const ownershipType = getFieldValue('ownershipType');
-    const isBusiness = ownershipType === OwnershipTypes.Business;
+    const [isShowMandateText, setIsShowMandateText] = useState(false);
```

Comment:
[optional]
I prefer to use `shouldShowMandateText`, `setShouldShowMandateText` for state variables naming. What do you think?

Reply:
Yeap, that would be better. I will rename it